app/src/main/java/com/github/houseorganizer/houseorganizer/panels/main_activities/CalendarActivity.java

```diff
@@ -46,13 +49,26 @@ protected void onCreate(Bundle savedInstanceState) {
         findViewById(R.id.calendar_screen_view_change).setOnClickListener(v -> {
             calendarAdapter = calendar.rotateCalendarView(this, calendarAdapter, calendarEvents);
             calendarAdapter.refreshCalendarView(this, currentHouse, "refreshCalendar:failureToRefresh", calendar.getView() == Calendar.CalendarView.MONTHLY);
+            yearMonth.setText(YearMonth.now().format(DateTimeFormatter.ofPattern("LLLL uuuu")));
```

Comment:
How is this pattern represented in the app? (LLLL uuuu)

Reply:
Its the title on top of the monthly calendar view (May 2022 for example), LLLL means the month and uuuu is the year